File: test/fixtures/r/post_xpost.r
library(httr)

headers = c(
  `Content-Type` = "application/x-www-form-urlencoded"
)

data = '{"keywords":"php","page":1,"searchMode":1}'

res <- httr::POST(url = "http://localhost:28139/api/xxxxxxxxxxxxxxxx", httr::add_headers(.headers=headers), body = data)

File: test/fixtures/clojure/post_xpost.clj
(require '[clj-http.client :as client])

(client/post "http://localhost:28139/api/xxxxxxxxxxxxxxxx" {:headers {"Content-Type" "application/x-www-form-urlencoded"}
                                                            :body "{\"keywords\":\"php\",\"page\":1,\"searchMode\":1}"})
